### PR TITLE
feature/LP-134-OutlinedTextfieldInput

### PR DESCRIPTION
**Pull request by AndresIvanMedinaH**
Task’s name
feature/LP-134-OutlinedTextfieldInput

Task’s link to navigate to jira
https://devaptivist.atlassian.net/browse/LP-134?atlOrigin=eyJpIjoiNzk0ZWY0ZmRkOWRlNDczOGI3YmVmMWZlMDdmZTAyNWYiLCJwIjoiaiJ9

General task’s description of what was added, deleted or updated in the code.
Task performed on Fix:
+change keyboard to alphanumeric using property KeyboardType.Text
+Added parameter Enabled to
+Changed name to something more generic LPOutlinedTextFieldInput

• Attachment file of how looks your code in the app or in the preview if it is necessary.

unit

unit2

### CODE REVIEW

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt [hunk 1]
```diff
@@ -147,8 +148,9 @@ fun LpOutlinedTextFieldMail(
  */
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun LpOutlinedTextFieldNumber(
+fun LpOutlinedTextFieldInput(
     modifier: Modifier,
+    enabled:Boolean,
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt]: Oh, please add a default value as it is in one of the description of your ticket:
`To have “enabled” option as parameter passed to function (“true” as the default value), so the user will be unable to edit/modify the text`

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt [hunk 2]
```diff
@@ -161,6 +163,7 @@ fun LpOutlinedTextFieldNumber(
             onValueChange(textFieldValue.text)
         },
         modifier = modifier.fillMaxWidth(),
+        enabled=enabled,
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt]: Please reformat code

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt [hunk 3]
```diff
@@ -147,8 +147,9 @@ fun LpOutlinedTextFieldMail(
  */
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun LpOutlinedTextFieldNumber(
+fun LpOutlinedTextFieldInput(
     modifier: Modifier,
+    enabled: Boolean,
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt:152]: This new variable needs to have a default value. It is a required updated of the ticket:
`To have “enabled” option as parameter passed to function (“true” as the default value), so the user will be unable to edit/modify the text`

Comment (AndresIvanMedinaH) [reply core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt:152]: fixing required update

**Pull request closed without merging**